### DISCUSSION TITLE
Show your templates for event notes

### DISCUSSION THREAD

**Discussion by machado-t** [YukiGasai/obsidian-google-calendar#158] [Show and tell]
I've just started using this plugin to create notes on events.
As of now, i'm using a very simple template that inserts some information in the frontmatter:
```
---
class: event
location: {{gEvent.location}}
start: {{gEvent.start.dateTime}}
---
```

Would you care to share yours?

**Comment by ShingHimYip** [comment]
I try to make the template work while create the event note manually  , but it didn't seem working , would you mind sharing how can u make it work ?
